templates/taxi/car_confirm_delete.html

```diff
@@ -6,5 +6,6 @@ <h1>Delete car?</h1>
     {% csrf_token %}
 
     <input type="submit" value="Yes" class="btn btn-danger">
+    <a href="{% url 'taxi:car-detail' pk=car.id %}" class="btn btn-secondary">Cancel</a>
```

Comment:
Ensure that the `car` context variable is passed to this template. This is essential for the URL tag to function correctly and provide the correct link for the 'Cancel' button.